src/plugins/scroll-snap.js

```diff
@@ -0,0 +1,13 @@
+import prefix from '../prefix'
+
+export default {
+  supportedProperty: (prop) => {
+    if (prop.substring(0, 11) === 'scroll-snap') {
+      if (prefix.js === 'ms') {
```

Comment:
you want to add prefix for IE only? Why not to exit early before any code needs to bee executed then? On line 2

Reply:
@kof, yes, prefix needed only, for IE/Edge. Didn't understood how i can this one code better:(

Reply:
This plugin seems to do nothing when browser is not IE, why not to do an early exit on line 2 then? prefix.js is already available there.

Reply:
Ah, no. It also should return value if browser is Safari. And should be prefixed, if IE.

Reply:
@kof, so that's why I wrote it. If no Safari support needed, there is no needed on this plugin

Reply:
may be, we need to add safari 11 to our test suite in `browsers.json`, to check this changes?